packages/ws-server/src/webSocketServer.ts

```diff
@@ -0,0 +1,106 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import Koa from 'koa';
+import jsonResp from '@hashgraph/json-rpc-server/dist/koaJsonRpc/lib/RpcResponse';
+import websockify from 'koa-websocket';
+import { Relay, RelayImpl, predefined } from '@hashgraph/json-rpc-relay';
+import { Registry } from 'prom-client';
+import pino from 'pino';
+
+const mainLogger = pino({
+    name: 'hedera-json-rpc-relay',
+    level: process.env.LOG_LEVEL || 'trace',
+    transport: {
+        target: 'pino-pretty',
+        options: {
+            colorize: true,
+            translateTime: true
+        }
+    }
+});
+
+const logger = mainLogger.child({ name: 'rpc-server' });
+const register = new Registry();
+const relay: Relay = new RelayImpl(logger, register);
+
+const app = websockify(new Koa(), {});
+const LOGGER_PREFIX = 'WebSocket:';
+
+app.ws.use((ctx) => {
+    ctx.websocket.id = '0x00000000000000000000000000000000';
+    // ctx.websocket.id = relay.subs().generateId();
```

Comment:
Presuming the comments are for future items.
If so you can remove them as they'll be added instead of leaving commented code